react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isClicked: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ isClicked: !this.state.isClicked });
+  }
+
+  render() {
+    const clicked = this.state.isClicked;
+    return (
+      <div id='react-toggle-switch'>
+        <div className={!clicked ? 'switch background' : 'switch background-active'}></div>
+        <div onClick={this.handleClick} className={!clicked ? 'slide slide-direction' : 'slide slide-direction-active'}></div>
+        <label>{!clicked ? 'OFF' : 'ON'}</label>
```

Comment:
These ternaries are more confusing than they need to be. If you remove the `!`, you can switch up the order of the results.

```js
clicked ? 'switch background-active' : 'switch background'
```